xdynamo/api.py

```diff
@@ -61,6 +61,7 @@ def get(
             fields: FieldNames | DefaultType | None = Default,
             allow_scan=False,
             consistent_read: bool | DefaultType = Default,
+            reverse: bool | DefaultType = Default
```

Comment:
I would just keep this simple, and not have a class-wide default for this.  You  can default this to `False` here and just pass it along just like your doing (with a default `False` value for each function you pass it into).